site/components/pages/Redesign/NextImage.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from "react";
+import Image, { ImageProps } from "next/image";
+
+import styles from "./index.module.css";
+
+// https://github.com/vercel/next.js/discussions/18739#discussioncomment-344932
```

Comment:
super weird I don't remember having this problem, I have done `fill` on quite a few images...